src/index.html

```diff
@@ -11,5 +14,9 @@
 </head>
 <body>
   <app-root></app-root>
+  <script src="https://code.jquery.com/jquery-3.3.1.slim.min.js" integrity="sha384-q8i/X+965DzO0rT7abK41JStQIAqVgRVzpbzo5smXKp4YfRvH+8abtTE1Pi6jizo" crossorigin="anonymous"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.7/umd/popper.min.js" integrity="sha384-UO2eT0CpHqdSJQ6hJty5KVphtPhzWj9WO1clHTMGa3JDZwrnQq4sF86dIHNDz0W1" crossorigin="anonymous"></script>
+  <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/js/bootstrap.min.js" integrity="sha384-JjSmVgyd0p3pXB1rRibZUAYoIIy6OrQ6VrjIEaFf/nJGzIxFDsf4x0xIM+B07jRM" crossorigin="anonymous"></script>
+
```

Comment:
Why you don't use the framework material-css included in the package ?

Reply:
Because I familiar with bootstrap and mdbootstrap so I use it.. And it gives a responsive web app. material CSS is also good but I prefer it..

Reply:
> Because I familiar with bootstrap and mdbootstrap so I use it.. And it gives a responsive web app. material CSS is also good but I prefer it..

Agreed, but when you are working on a project, you have to follow / use the tools and utilities that is currently being used in the project.

So please update your code to use the tools and utilities that this project is using and commit it to your branch. The PR will be auto-updated once you commit.

Reply:
so now i totally remove bootstrap scripts ?

Reply:
> so now i totally remove bootstrap scripts ? and update all components in bootstrap to material design ? suggest me what can I do now..

Unfortunately, yes. Use the project setup @maaxow is using. You will also get to learn/master different frameworks. :slightly_smiling_face:

Reply:
ohhk got it I can change in this remove all bootstrap script and add angular-material design.. 
**COOL**

Reply:
Thanks guys. It's like that I guess the real contributing 😄 

Normally, Material is already init in the project. Do you run the `npm install` in your project ?